gremlin-scala/src/main/scala/gremlin/scala/GremlinScala.scala

```diff
@@ -673,6 +673,9 @@ case class GremlinScala[End, Labels <: HList](traversal: GraphTraversal[_, End])
   // NUMBER STEPS END
   // -------------------
 
+  def V(vertexIdsOrElements: Any*)(implicit ev: End <:< Vertex): GremlinScala[Vertex, Labels] =
```

Comment:
Just a note: I was wondering if we actually need `End <:< Vertex`, so I removed the implicit and played with the test to see what would happen... turns out it runs fine, but the results are quite different with this test:

```
      graph.V().has(Name, "marko").as(a).outE
        .V().has(Name, "vadas").as(b)
        .addE(Knows).from(a).to(b).property(StartTime, 2010)
        .iterate
```
results: 
```
can be used mid traversal *** FAILED ***
[info]   List("marko", "marko", "marko", "vadas", "vadas", "vadas") did not equal List("marko", "vadas") (TraversalSpec.scala:596)
```